egs/rm/s5/local/chain/tuning/run_tdnn_wsj_rm.sh

```diff
@@ -0,0 +1,192 @@
+#!/bin/bash
```

Comment:
This is not a softlink. Git will show it as softlink if it is. There is already local/chain/run_tdnn_wsj_rm.sh which is a softlink to _1a.